<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UC_DlgClass</class>
 <widget class="QDialog" name="UC_DlgClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>269</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UC_Dlg</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="1">
      <widget class="QPushButton" name="Rawimage">
       <property name="styleSheet">
        <string notr="true"> border:2px groove gray;border-radius:6px;padding:4px 48px;
</string>
       </property>
       <property name="text">
        <string>浏览</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>原始影像：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLineEdit" name="ImageEdit">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"> border:2px groove gray;border-radius:4px;padding:2px 4px;
background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="1" column="0">
      <widget class="QLineEdit" name="Edit_output">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"> border:2px groove gray;border-radius:4px;padding:2px 4px;
background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pb_output">
       <property name="styleSheet">
        <string notr="true"> border:2px groove gray;border-radius:6px;padding:4px 48px;
</string>
       </property>
       <property name="text">
        <string>浏览</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>输出文件位置：</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>匀色方法：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>最小二乘拟合</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>直方图匹配</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Wallis滤波</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="styleSheet">
        <string notr="true"> border:2px groove gray;border-radius:6px;padding:4px 48px;
</string>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ok">
       <property name="styleSheet">
        <string notr="true"> border:2px groove gray;border-radius:6px;padding:4px 48px;
</string>
       </property>
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
